styleguide/source/_patterns/03-organisms/column-teaser.json

```diff
@@ -0,0 +1,55 @@
+{
+  "items": [
+    {
+      "patternLab": true,
+      "related": true,
+      "link": {
+        "title": "Link to something",
+        "href": "#",
+        "text": "Lorem ipsum dolor sit amet.",
+        "class": "ama__link-black",
+        "target": "_self"
+      },
+      "image": {
+        "alt": "alt text",
+        "src": "https://ipsumimage.appspot.com/279x186?l=3:2|279x186&s=36",
+        "height": "186",
+        "width": "279"
+      }
+    },
+    {
+      "patternLab": true,
+      "related": true,
+      "link": {
+        "title": "Link to something",
+        "href": "#",
+        "text": "Lorem ipsum dolor sit amet.",
+        "class": "ama__link-black",
```

Comment:
This should be `ama__link--black` since black is a modifier. Also I think black is the default so you may not need the class at all.